data/human/sales.txt

```diff
@@ -153,6 +153,7 @@ outfitter "Basic Outfits"
 
 outfitter "Common Outfits"
 	"Energy Blaster"
+	"X1100 Ion Reverse Thruster"
```

Comment:
I think this should be a Syndicate special, not a common outfit, just like the reverse atomic is a Deep Sky special.  Not even the Navy shipyards get access to the atomic reverse thruster.

Since I cannot add a suggestion to a line you didn't change, see https://github.com/Zitchas/endless-sky/pull/95

Reply:
Well, I kind of like having the basic ionic reverse thrusters being a bit more common than that. They get little enough love as it is, at least giving people the option to use them as a "temporary stop gap" until they can get a better one means they might see *some* use.